app/models/space.js

```diff
@@ -2,38 +2,51 @@ import Ember from 'ember';
 
 export default Ember.Object.extend({
 
-  name      : '',
-  ircServer : {
-    hostname: 'irc.freenode.net',
-    port: 6667,
-    secure: false,
+  name      : null,
+  protocol  : 'IRC',
+  server : {
+    hostname: null,
+    port: 7000,
+    secure: true,
     username: null,
     password: null,
     nickname: null,
     nickServ: {
       password: null
     }
   },
-  channels  : null,
-  users     : null,
+  channels   : null, // Channel instances
+  channelList: [],   // Bookmarked channel names
```

Comment:
You should never set an `Ember.Object`'s property to an array like this, because then the same array will be shared by all instances of the object.

You should rather set the initial empty array in the `init()` method instead.

See also https://guides.emberjs.com/v2.6.0/object-model/classes-and-instances/#toc_initializing-instances